### PR TITLE
Gce ui api representation

### PR DESCRIPTION
**Pull request by kzvyahin**
{{pytest : cfme/tests/cloud/test_provisioning.py -k "test_provision_from_template" --use-provider="gce_central"}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 941059220400233f4dba32d97be9a36efb2e2ab4:

Everything seems all right from lint perspective :smile: .

Commit inspection report:
* a2ff48d0ce6f41a8a79fea014a277cf4e83d8951
  * Merge commit detected!!! :bangbang:

**The PR might not be acceptable in this state** :rage4:.
There is a merge commit in the branch which can cause some problems during squashing. Please take **EXTRA care** when squashing!
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

File: cfme/provisioning.py [hunk 1]
```diff
@@ -188,6 +189,9 @@ def select_security_group(sg):
             ('hardware_monitoring', {
                 version.LOWEST: ui.Select('select#hardware__monitoring'),
                 '5.5': AngularSelect('hardware__monitoring')}),
+            ('boot_disk_size', {
+                version.LOWEST: ui.Select('select#hardware__boot_disk_size'),
+                '5.5': AngularSelect('hardware__boot_disk_size')}),
```
Comment (mfalesni) [inline cfme/provisioning.py]: At this point we don't care about anything less than 5.5 so you can remove the ui.Select reference I think.

File: cfme/cloud/instance.py [hunk 2]
```diff
@@ -361,6 +361,77 @@ def power_control_from_provider(self, option):
...
+    # CFME power states
+    STATE_ON = "on"
+    STATE_OFF = "off"
+    STATE_SUSPENDED = "suspended"
+    STATE_TERMINATED = "terminated"
+    STATE_ARCHIVED = "archived"
+    STATE_UNKNOWN = "unknown"
+
+    def create(self, email=None, first_name=None, last_name=None, availability_zone=None,
+               instance_type=None, cancel=False, cloud_network=None,
```
Comment (jkrocil) [inline cfme/cloud/instance.py]: We usually keep the `cancel` attribute as the last one to separate between data that get filled into the form and the final action (cancel / submit).

File: cfme/provisioning.py (hunk 1 above)
Comment (jkrocil) [inline cfme/provisioning.py]: Yep. We don't care about `5.5` anymore. Let's just use the `AngularSelect` directly. Feel free to update the other `5.5` version picks as well.

File: cfme/tests/cloud/test_provisioning.py [hunk 3]
```diff
@@ -433,3 +448,8 @@ def _finish_method():
         if "volume_id" in locals():  # To handle the case of 1st or 2nd assert
             if provider.mgmt.volume_exists(volume_id):
                 provider.mgmt.delete_volume(volume_id)
+
+
+def test_temp():
```
Comment (jkrocil) [inline cfme/tests/cloud/test_provisioning.py]: I assume this will go away? Same with the `get_mgmt` import at the top of this module.

**Comment by dajoRH** [comment]
Lint report for commit 93d50cfc6c74df547f653f934b5d79923ea1fceb:

`cfme/tests/cloud/test_provisioning.py`:
- :red_circle: Line 20:1: **F401** *'get_mgmt' imported but unused*
- :heavy_exclamation_mark: Line 451:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .

Commit inspection report:
* a2ff48d0ce6f41a8a79fea014a277cf4e83d8951
  * Merge commit detected!!! :bangbang:

**The PR might not be acceptable in this state** :rage4:.
There is a merge commit in the branch which can cause some problems during squashing. Please take **EXTRA care** when squashing!
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 9c7f10b124c64c6acbdf34c03200bbfae0e64312:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c662a3a89580ae09c31a77735daa59c1f848bca4:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/cloud/test_provisioning.py [hunk 4]
```diff
@@ -66,8 +71,11 @@ def testing_instance(request, setup_provider, provider, provisioning, vm_name):
 
 
 @pytest.fixture(scope="function")
-def vm_name(request):
-    vm_name = 'test_image_prov_{}'.format(fauxfactory.gen_alphanumeric())
+def vm_name(request, provider):
+    if provider.key == 'gce_central':
```
Comment (jkrocil) [inline cfme/tests/cloud/test_provisioning.py]: We cannot decide this based on `provider.key`, it has to be done the same way as above, using `isinstance`.

**Comment by dajoRH** [comment]
Lint report for commit 82a9280dfaf435901ed34181477d7134378215dc:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**